app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -90,6 +97,20 @@ class BookmarksActivity : DuckDuckGoActivity() {
         })
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(bookmark_activity_menu, menu)
+        val searchItem = menu?.findItem(action_search)
+        val searchView = searchItem?.actionView as SearchView
+        searchView.setHintTextColor(Color.GRAY)
```

Comment:
Let's delete this and use our app themes to set the color. In themes.xml add the following to the bottom of the main app theme `AppTheme` style:
```
<item name="autoCompleteTextViewStyle">@style/AutoCompleteTextViewStyle</item>
```

And then add the definition to the bottom of the themes file:
```
<style name="AutoCompleteTextViewStyle" parent="Widget.AppCompat.AutoCompleteTextView">
    <item name="android:textCursorDrawable">@drawable/text_cursor</item>
    <item name="android:textColorHint">?attr/omnibarHintColor</item>
    <item name="android:fontFamily">sans-serif-medium</item>
</style>
```